src/absil/illib.fs

```diff
@@ -942,26 +942,18 @@ type LayeredMultiMap<'Key,'Value when 'Key : equality and 'Key : comparison>(con
 module Shim =
 
     open System.IO
-    [<AbstractClass>]
-    type FileSystem() = 
-        abstract ReadAllBytesShim: fileName:string -> byte[] 
-        default this.ReadAllBytesShim (fileName:string) = 
```

Comment:
This default implementation was never used by Visual F# Tools and was not present in FCS